### ISSUE TITLE
Vagrant config doesn't include conntrack (needed by kube-proxy)

### ISSUE THREAD

**Issue by fasaxc** [projectcalico/calico#1058]
Reported by @chino: https://github.com/projectcalico/calico/issues/1055#issuecomment-324848493

Our vagrantfile installs kube-proxy without all of its dependencies.  For example, kube-proxy requires the `conntrack` tool to be availabel in order to clean out stale conntrack entries.

## Current Behavior
kube-proxy unable to clean up conntrack entries, errors in logs.

## Possible Solution
Either install the deps one by one or, if available, use a kube-proxy container image instead.

## Steps to Reproduce (for bugs)
More details of set-up in #1055 

## Context
In #1055, this resulted in blackholed UDP traffic.

## Your Environment

* Calico version: 2.5.0
* Orchestrator version (e.g. kubernetes, mesos, rkt): k8s v1.7.0 (as shipped in our vagrant set-up)
* Operating System and version: 
* Link to your project (optional):

**Comment by chino** [comment]
I do think it would be great to run kubelet/others in containers.  Perhaps using BootKube or an installer that wraps it could be cool too?

To fix the issue I modified the `master/node-config.yaml` files by:
- bumping k8s to v1.7.4 (the conntrack fix was in v1.7.1 I believe)
- bumping cni to v0.6.0 (not sure if that was needed)
- making sure `conntrack` is in kube-proxy's $PATH
- Also added `--v 3` to kube-proxy so I could see conntrack related actions in the logs
  - https://github.com/kubernetes/kubernetes/blob/master/pkg/proxy/iptables/proxier.go#L972

At that point I noticed my existing blackholed sessions immediately got fixed!

Note: Statically compiling the `conntrack` util turned out to be quiet the journey... As a quick hack / proof I simply created a script with the following contents which worked:
```
#!/usr/bin/env bash
docker run --privileged --net=host -i --rm --name=conntrack claesjonsson/conntrack "$@"
```

**Comment by caseydavenport** [comment]
We no longer maintain a vagrantfile, so going to close this.
